### ISSUE TITLE
Button to close edit options for task

### ISSUE THREAD

**Issue by KeepFloyding** [alainm23/planify#666]
**Is your feature request related to a problem? Please describe.**
When I click on a task to edit it, I am presented with an edit dialogue where I can add notes, schedule etc. However there is no button that I can click to exit this dialogue; instead I have to press the ESC key. 

**Describe the solution you'd like**
It would be great to have a 'X' button to close the edit dialogue for new users.

**Additional context**
This is on Fedora 33

**Comment by koudi** [comment]
I'm on fedora and there is icon in right upper corner.

Did you install elementary theme/icons?

See readme:

    You should install elementary-theme and elementary-icon-theme to be a complete experience.

**Comment by alainm23** [comment]
This icon is available, how did you install Planner?
![image](https://user-images.githubusercontent.com/33765137/103929719-b3dbdf80-50eb-11eb-8174-e653ab9d49d3.png)

**Comment by BeatLink** [comment]
The "Leave Fullscreen" icon shown in the top right in @alainm23's comment above would be the button used to minimize a task. It does the same action as the ESC key.

**Comment by KeepFloyding** [comment]
Hi, sorry for the late response. Many thanks for the replies. @koudi 's comment fixed it for me. I needed to install the elementary theme and icon-theme to get the images shared above. With regards to the installation procedure, I just did a simple `sudo dnf install elementary-planner`. Would it be worthwhile including the installation of the the elementary themes by default when installing planner? Many thanks!
